pkg/config/prometheus_config.go

```diff
@@ -13,11 +13,20 @@ const (
 	CfgPrometheusProcessMetrics = "prometheus.processMetrics"
 	// include promhttp metrics
 	CfgPrometheusPromhttpMetrics = "prometheus.promhttpMetrics"
+	// "whether the plugin should write a Prometheus 'file SD' file"
+	CfgPrometheusFileServiceDiscoveryEnabled = "prometheus.fileServiceDiscovery.enabled"
+	// the path where to write the 'file SD' file to
+	CfgPrometheusFileServiceDiscoveryPath = "prometheus.fileServiceDiscovery.path"
+	// the target to write into the 'file SD' file
+	CfgPrometheusFileServiceDiscoveryTarget = "prometheus.fileServiceDiscovery.target"
```

Comment:
Would it make sense to create a slice with several targets?

Reply:
no